chain/ethereum/src/block_ingestor.rs

```diff
@@ -115,6 +147,10 @@ where
                                     LogCode::BlockIngestionStatus
                                 };
                                 if distance > 0 {
+                                    self.ingestor_metrics.observe_blocks_synced(
```

Comment:
This is not the right place IMHO. At this point we have't written the block to our db yet, so as far as our system is concerned, we are not up to date with the chain yet.

Reply:
I think the right place would be an `inspect` here: https://github.com/graphprotocol/graph-node/pull/1403/files#diff-dc8c393601039a99522fbd9bdd37b8dbR233-R235

Reply:
I'm not sure I follow how that would work. Also generally, it is tricky because `ingest_blocks()` will get called again for any missing blocks.  

What about observing the `chain_head_number` from the `ChainHeadUpdateListener`?

Reply:
I like that idea!